### PR TITLE
New architecture

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,11 @@
     "lint:fix": "eslint --ext .js --fix ./"
   },
   "dependencies": {
-    "mhub": "^2.0.0"
+    "@first-lego-league/ms-correlation": "^1.0.2",
```
Comment (2roy999) [inline package.json]: optional

File: package.json [hunk 2]
```diff
@@ -19,7 +19,11 @@
...
+    "@first-lego-league/ms-logger": "^2.0.1",
```
Comment (2roy999) [inline package.json]: optional

File: package.json [hunk 3]
```diff
@@ -19,7 +19,11 @@
...
+    "detect-node": "^2.0.4",
```
Comment (2roy999) [inline package.json]: remove

**Comment by idanstark42** [comment]
@2roy999 can you take a look here? This is failing because of the space in `Program Files` folder name, but I can't figure out why it's using it.
It should be noted that this only happend with `nyc`.

**Pull request merged**